eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/ExcludeStructuralVariantsProcessorTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.pipeline.steps.processors;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.commons.core.models.IVariant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class ExcludeStructuralVariantsProcessorTest {
+
+    private static final String ALT_WITH_SINGLE_BASE = "A";
+
+    private static final String ALT_WITH_MULTI_BASE = "AT";
+
+    private static final String ALT_WITH_EMPTY_ALLELE = "";
+
+    private static final String[] ALTS_WITH_SYMBOLIC_INDEL = new String[]{"<DEL>", "<INS>", "<DUP>", "<INV>", "<CNV>"
+            , "<DUP:TANDEM>", "<DEL:ME:C1#>", "<INS:ME:C1#>"};

```

Comment:
Be consistent with the formatting, we don't place commas at the beginning of a line.